src/core.js

```diff
@@ -50,7 +50,10 @@ export const getHeaderValue = (property, obj) => {
 export const elementOrEmpty = (element) => element || element === 0 ? element : '';
 
 export const joiner = ((data,separator = ',') =>
- data.map((row, index) => row.map((element) => "\"" + elementOrEmpty(element) + "\"").join(separator)).join(`\n`)
+ data.map((row, index) => {
+  if(row == null) return null
```

Comment:
Would this not be neater if you used `filter` on the data? instead of returning null in a map @NandoTheessen

Reply:
@mccabemj the difference between the two is that filtering the data removes the empty line from the output, whereas returning `null` in the map function leaves the line in output, albeit as an empty line. 

I am leaning towards filtering null and undefined rows before mapping as you suggested so that they don't appear at all, but I am keen to hear what other users of this library would want

Reply:
Yeah @mriccid in the issue @NandoTheessen raised, I was leaning towards leaving the blank lines in. I think it would be neater to omit them.

@NandoTheessen you may be best placed to answer this, as you have source data with null lines. What would you expect?

It would be quite interesting to allow blank lines. I can see situations where people would want reports that have blank lines to separate groupings of data. Perhaps we could even add this to the documentation as a feature.

Reply:
Honestly, having no blank lines would be the most intuitive to me. 
The way I've encountered this is when working with firebase (where deleted entries would return null / undefined in the array).

The user doesn't even know that that is going on behind the scenes. For them expected behaviour would indeed be no empty lines.
I'll change my PR to reflect that.